spec/system/candidate_interface/course_selection/candidate_can_view_degree_requirements_and_guidance_for_selected_courses_spec.rb

```diff
@@ -0,0 +1,133 @@
+
+    when_i_change_my_degree_grade_to_a_pass
+    and_i_visit_the_course_choice_review_page
+    then_i_can_course_choices_choices_with_relevant_guidance_for_all_courses
+  end
+
+  def given_i_am_signed_in
+    create_and_sign_in_candidate
+  end
+
+  def and_i_have_three_chosen_courses_with_different_requirements
+    course_option1 = create(:course_option, course: create(:course, :open_on_apply, degree_grade: 'two_one'))
+    course_option2 = create(:course_option, course: create(:course, :open_on_apply, degree_grade: 'two_two'))
+    course_option3 = create(:course_option, course: create(:course, :open_on_apply, degree_grade: 'third_class'))
+    @choice1 = create(
+      :application_choice,
+      status: :unsubmitted,
+      course_option: course_option1,
+      application_form: current_candidate.current_application,
+    )
+    @choice2 = create(
+      :application_choice,
+      status: :unsubmitted,
+      course_option: course_option2,
+      application_form: current_candidate.current_application,
+    )
+    @choice3 = create(
+      :application_choice,
+      status: :unsubmitted,
+      course_option: course_option3,
+      application_form: current_candidate.current_application,
+    )
+  end
+
+  def when_i_add_a_two_two_degree
+    visit candidate_interface_new_degree_path
+
+    choose 'UK degree'
+    fill_in 'Type of degree', with: 'BA'
+    click_button t('save_and_continue')
+
+    fill_in 'What subject is your degree?', with: 'Doge'
+    click_button t('save_and_continue')
+
+    fill_in 'Which institution did you study at?', with: 'University of Much Wow'
+    click_button t('save_and_continue')
+
+    expect(page).to have_content('Have you completed your degree?')
+    choose 'Yes'
+    click_button t('save_and_continue')
+
+    choose 'Lower second-class honours (2:2)'
+    click_button t('save_and_continue')
+
+    year_with_trailing_space = '2006 '
+    fill_in t('page_titles.what_year_did_you_start_your_degree'), with: year_with_trailing_space
+    click_button t('save_and_continue')
+    year_with_preceding_space = ' 2009'
+    fill_in t('page_titles.what_year_did_you_graduate'), with: year_with_preceding_space
+    click_button t('save_and_continue')
+    choose t('application_form.completed_radio')
+    click_button t('continue')
+  end
+
+  def and_i_visit_the_course_choice_review_page
+    visit candidate_interface_course_choices_review_path
+  end
+
+  def then_i_can_course_choices_choices_with_relevant_guidance_for_one_course
+    within "#course-choice-#{@choice1.id}" do
+      expect(page).to have_content('Degree requirements')
+      expect(page).to have_content('2:1 degree or higher (or equivalent)')
+      expect(page).to have_content('You said you have a 2:2 degree.')
+      expect(page).to have_content("You can:\nfind a course that has a lower degree requirement contact the provider to see if they will still consider your application")
```

Comment:
is this not the opposite of what we would expect? i.e. only the 4d course should have the content on contacting the provider?